### PR TITLE
Add PendingIntentCompat helper methods.

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**
Add helper methods to `PendingIntentCompat` to avoid issues on API levels < 23.

### CODE REVIEW

File: app/src/main/java/com/github/libretube/compat/PendingIntentCompat.kt [hunk 1]
```diff
@@ -1,20 +1,30 @@
 package com.github.libretube.compat
 
 import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
 import android.os.Build
 
+// TODO: Use AndroidX's PendingIntentCompat class instead once it becomes available.
```
Comment (Bnyro) [inline app/src/main/java/com/github/libretube/compat/PendingIntentCompat.kt:8]: Have there been any announcements for a `PendingIntentCompat` being added to the `androidx` library in the near future recently? (just out of interest)

**Review by Bnyro — APPROVED** [review]

File: app/src/main/java/com/github/libretube/compat/PendingIntentCompat.kt (hunk 1 above)
Comment (Isira-Seneviratne) [reply app/src/main/java/com/github/libretube/compat/PendingIntentCompat.kt:8]: It's there in the repo: https://github.com/androidx/androidx/blob/androidx-main/core/core/src/main/java/androidx/core/app/PendingIntentCompat.java

Comment (Bnyro) [reply app/src/main/java/com/github/libretube/compat/PendingIntentCompat.kt:8]: So it's probably going to be added with the next version bump of `androidx.core`?

Comment (Isira-Seneviratne) [reply app/src/main/java/com/github/libretube/compat/PendingIntentCompat.kt:8]: Seems like it.

**Pull request merged**